pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/tasks/SourceDistTask.java

```diff
@@ -35,7 +35,11 @@ public void processResults(ExecResult execResult) {
     @OutputFile
     public File getSdistOutput() {
         Project project = getProject();
-        return new File(getDistDir(), String.format("%s-%s.tar.gz", project.getName(), project.getVersion()));
+        return new File(getDistDir(), 
```

Comment:
Can you break the line after the opening parenthesis, to have the parameters of `File()` aligned at the same indent level.